code/thrive-design-lightbox.js

```diff
@@ -6,123 +6,129 @@ $(function () {
         $(this).wrap('<a href="' + src + '" class="big" rel="' + rel + '"></a>');
     });
 
-    // As A jQuery Plugin -->
-    $('.gallery a').simpleLightbox({
-        // default source attribute
-        sourceAttr: 'href',
 
-        // shows fullscreen overlay
-        overlay: true,
+    if (!!($('.gallery').html()) &&
+    !!($('.gallery a').attr('href'))) {
```

Comment:
Pretty much; without that code it was breaking all the JS on the sticker sheet because it was trying to get the href attribute on a link in a container with the .gallery class, but there was no element with the .gallery class so it just threw errors.